src/plugins/stepsform/stepsform.js

```diff
@@ -227,3 +227,80 @@ $document.on( "timerpoke.wb " + initEvent, selector, init );
 wb.add( selector );
 
 } )( jQuery, window, document, wb );
+
+( function( $, document ) {
+"use strict";
+
+// Fetch page language and set variables accordingly
+let relpreposition = " of ",
+	progressLabel = "Questionnaire progress:";
+
+// Define French progress label
+if ( wb.lang === "fr" ) {
+	relpreposition = " de ";
+	progressLabel = "Progression du questionnaire : ";
+}
+
+//Detect the enhancement of the quiz
+var quizSelector = ".provisional.wb-steps.quiz";
+
+//Initialize all instances
+var init = function( e ) {
+	let $instance = $( e.currentTarget );
+
+	// Calculate number of questions
+	let numQuestion = $( ".steps-wrapper", $instance ).length;
+
+	// Addition to UI (Ex: progress bar)
+	if ( !$.contains( $instance, "progress" ) ) {
+		$( "form", $instance ).prepend( "<label class='full-width'><span class='wb-inv'>" + progressLabel + "</span><progress class='progressBar' max='" + numQuestion + "'></progress><p class='progressText' role='status'></p></label>" );
+	}
+
+	hideOtherSteps( e );
+};
+
+var hideOtherSteps = function( e ) {
+
+	// Get wb-steps component
+	let steps,
+		currentElement = e.currentTarget;
+
+	if ( currentElement.classList.contains( "quiz" ) && currentElement.classList.contains( "wb-steps" ) ) {
+		steps = currentElement;
+	} else {
+		steps = $( currentElement ).parentsUntil( quizSelector ).parent().get( 0 );
+	}
+
+
+	// Check if the instance is not found
+	if ( !steps || steps instanceof HTMLDocument ) {
+		return;
+	}
```

Comment:
This would not be needed inside the other init.

Reply:
Resolved in latest commitr